### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by alicehsing**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,3 +1,14 @@
+import { Route, Switch } from 'react-router-dom';
+import FuturamaList from './views/Futurama/List';
+import './App.css';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <Switch>
+      <Route path="/">
+        <h1>Futurama Compendium</h1>
```
Comment (BenFrancka) [inline src/App.jsx]: This is fine for a small app like this, but remember that the cool thing about react is that you can make re-useable components, so I would probably make this `h1` a `Header` component

File: src/tests/futuramaData.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904145/leela.png',
+  },
+  {
+    character: 'Professor Farnsworth',
+    quote: 'You may feel a slight stinging sensation ...all of you.',
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904014/farnsworth.png',
+  },
+];
```
Comment (BenFrancka) [inline src/tests/futuramaData.js]: It is a pretty common naming convention to keep your mock data for testing in a `fixtures` folder.

Comment (alicehsing) [reply src/tests/futuramaData.js]: I see. Thanks for clarifying!

File: src/App.jsx (hunk 1 above)
Comment (alicehsing) [reply src/App.jsx]: Totally makes sense.

**Pull request merged**